### ISSUE TITLE
avs-device-sdk complie fail

### ISSUE THREAD

**Issue by s00275669** [Kitt-AI/snowboy#388]
home/pi_kitt/avs-device-sdk/KWD/KittAi/src/KittAiKeyWordDetector.cpp:122:78: error: narrowing conversion of ‘(((std::chrono::duration<long long int, std::ratio<1ll, 1000ll> >::rep)(audioFormat.alexaClientSDK::avsCommon::utils::AudioFormat::sampleRateHz / 1000u)) * msToPushPerIteration.std::chrono::duration<_Rep, _Period>::count<long long int, std::ratio<1ll, 1000ll> >())’ from ‘std::chrono::duration<long long int, std::ratio<1ll, 1000ll> >::rep {aka long long int}’ to ‘size_t {aka unsigned int}’ inside { } [-Werror=narrowing]
         m_maxSamplesPerPush{(audioFormat.sampleRateHz / HERTZ_PER_KILOHERTZ) * msToPushPerIteration.count()}

**Comment by s00275669** [comment]
we run the avs in pi,when use sensory is ok,but use kitt_ai,compile err

**Comment by cucubits** [comment]
Have you found any solution for this? I think I ran into the same issue. I don't have my logs anymore as I reinstalled AVS SDK with the default Sensory steps and it worked fine.

Since AVS SDK documentation provides exact install guide only with Sensory, I would love to see a bit more detailed step by step guide for rpi on getting it installed with Kitt.ai). Maybe just the specific dependencies needed, exact package needed to download, install and then activate/integrate with the SDK.
